roles/katello_repositories/tasks/staging_repos.yml

```diff
@@ -33,6 +33,7 @@
     name: pulp-koji
     description: "Pulp {{ katello_repositories_version }} Koji Repository"
     baseurl: "http://koji.katello.org/releases/yum/katello-{{ katello_repositories_version }}/pulp/el{{ ansible_distribution_major_version }}/x86_64/"
+    baseurl: "https://repos.fedorapeople.org/repos/pulp/pulp/{{ kaatello_repositories_pulp_release }}/{{ katello_repositories_pulp_version }}/{{ ansible_distribution_major_version }}Server/x86_64/"
```

Comment:
Shouldn't ```x86_64``` be ```$basearch```?

Reply:
It can be, nothing else in this file uses it though, so I am being consistent.